code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -41,32 +42,112 @@
 /obj/item/weapon/reagent_containers/blood/attack(mob/living/carbon/human/M as mob, mob/living/carbon/human/user as mob)
 	if (user == M && (user.mind.vampire))
 		if (reagents.get_reagent_amount("blood"))
-			user.visible_message("\red [user] raises \the [src] up to \his mouth and bites into it.", "\blue You raise \the [src] up to your mouth and bite into it, starting to drain its contents.")
+			user.visible_message("\red [user] raises \the [src] up to \his mouth and bites into it.", "\blue You raise \the [src] up to your mouth and bite into it, starting to drain its contents.<br>You need to stand still.</span>")

```

Comment:
Broken HTML here, a stand alone `</span>` tag. Replace with:
`user.visible_message("<span class='warning'>[user] raises \the [src] up to \his[user] mouth and bites into it.</span>", "<span class='notice'>You raise \the [src] up to your mouth and bite into it, starting to drain its contents.<br>You need to stand still.</span>")`

also, I added param `[user]` to the macro `\his`. The macro should eat it up and display the proper pronoun.